debugtools/DDR_VM/src/com/ibm/j9ddr/vm29/tools/ddrinteractive/commands/ShrCCommand.java

```diff
@@ -952,6 +979,8 @@ private void dbgShrcPrintAllStats(PrintStream out, J9JavaVMPointer vm, J9SharedC
 					++numCacheletsNoSegments;
 				}
 				++numCachelets;
+			} else if (itemType.eq(TYPE_PREREQ_CACHE)) { 
+				/* TODO Add support for TYPE_PREREQ_CACHE */
```

Comment:
This reference to `TYPE_PREREQ_CACHE` with trigger a `NoSuchFieldError` when accessing older core files; adding this field will fix that:
```
private static final long TYPE_PREREQ_CACHE = J9ConstantHelper.getLong(ShcdatatypesConstants.class, "TYPE_PREREQ_CACHE", -1);
```